docs/console.html

```diff
@@ -0,0 +1,804 @@
+var user = "jsmith", authenticated = true, authorized = true;
+// Top-level group
+console.group("Authenticating user '%s'", user);
+if (authenticated) {
+    console.log("User '%s' was authenticated", user);
+    // Start nested group
+    console.group("Authorizing user '%s'", user);
+    if (authorized) {
+        console.log("User '%s' was authorized.", user);
+    }
+    // End nested group
+    console.groupEnd();
+}
+// End top-level group
+console.groupEnd();
+console.log("A group-less log trace.");
+</code></pre>
+                </figure>
+                <figure>
+                    <img src="console-files/nestedgroup.png" alt="Nested console group output"/>
+                    <figcaption>
+                        How nested groups are displayed.
+                    </figcaption>
+                </figure>
+            </figure>
+
+            <p>
+                When you are using groups heavily it can be very useful to not see everything as it happens.
+                For these times you can automatically collapse groups by calling <code><a href="console-api#consolegroupcollapsed">groupCollapsed()</a></code> instead of <code>group()</code>.
+            </p>
+
+            <figure>
+                <figcaption>console.groupCollapsed() usage</figcaption>
+                <figure>
+                    <figcaption>Example code:</figcaption>
+                    <pre class="prettyprint"><code class="language-javascript">
+console.groupCollapsed("Authenticating user '%s'", user);
+if (authenticated) {
+    ...
+}
+console.groupEnd();                     
+</code></pre>
+                </figure>
+                <figure>
+                    <img src="console-files/groupcollapsed.png" alt="Initially collapsed group" />
+                    <figcaption>
+                        GroupCollapsed output.
+                    </figcaption>
+                </figure>
+            </figure>
+        </section>
+
+        <section>
+            <h3 id="string-substitution-and-formatting">String substitution and formatting</h3>
+
+            <p>
+                The first parameter passed to any of the logging methods may contain one or more <i>format specifiers</i>.
+                A format specifier consists of a <code>%</code> symbol followed by a letter that indicates the formatting that should be applied to the value.
+                The specifier is a placeholder which is substituted in order by the following parameters.
+            </p>
+
+            <figure>
+                <figcaption>
+                    The following example uses the string and digit formatters to insert values into the output string.
+                </figcaption>
+                <pre class="prettyprint"><code class="language-javascript">
+                    console.log("%s has %d points", "Sam", "100");
+</code></pre>
+                This would result in <samp>Sam has 100 points</samp> being logged to the console.
```

Comment:
The actual result is:

> Sam has NaN points

It does work if the last param is not a string:

`console.log("%s has %d points", "Sam", 100);`